src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -516,6 +520,11 @@ std::vector<std::string> disabledTestPatterns() {
         retVector.emplace_back(R"(smoke_CompareWithRefs_4D.*/EltwiseLayerCPUTest.*Sub_secondary.*INFERENCE_PRECISION_HINT=f16.*FakeQuantize.*enforceSnippets=1.*)");
         retVector.emplace_back(R"(smoke_Reduce.*/ReduceCPULayerTest.*axes=\((0.1|1)\).*Prod_KeepDims.*INFERENCE_PRECISION_HINT=f16.*)");
         retVector.emplace_back(R"(smoke_ConvertRangeSubgraphCPUTest/ConvertRangeSubgraphCPUTest\.CompareWithRefs.*Prc=f16.*)");
+        // skip after convert precision migrate
```

Comment:
Could you please provide details of that issue?

Reply:
these test cases ir is as follows:
![1](https://github.com/openvinotoolkit/openvino/assets/56590087/a53fdd37-7aa7-4088-892e-ad361e1cdd7b)
by default master branch, the final graph will be : Reorder + Transpose(f16) + Reorder:
![2](https://github.com/openvinotoolkit/openvino/assets/56590087/fdb180dd-db86-4b54-9c00-95bc57419dca)
with Migrate pr: the graph after 'ConvertPrecision' PASS will be Convert + Transpose(f16) + Convert:
![3](https://github.com/openvinotoolkit/openvino/assets/56590087/6712ed46-ba14-4f12-be64-2e569fff05ba)
and the following 'ov::pass::TransposeSinking' will change the order of 'Transpose(f16) + Convert', so the final graph will be Convert +  Convert + Transpose(f32):
![4](https://github.com/openvinotoolkit/openvino/assets/56590087/2e86bae8-57de-4ae4-9f73-772407aaaa75)

this will generte 'primTypeCheck(primType) primType is unexpected : unknown_f32 Expected : unknown_f16' error.

Reply:
@itikhono Couldyou please take a look on this comment? Is it expected behavior of TransposeSinking logic?
Like after that pass we need to execute Transpose in higher precision (fp32 instead of fp16).

Reply:
Hi, @itikhono :  from my understanding, since the following 'Output' Node precision of 'Transpose' Node is f32, so  this 'Transpose' Node precision changes to f32 to keep algin with it seems make sense, do you have any suggestions about this? thanks

Reply:
Behavior within the current approach is correct. The goal of the algorithm is to reduce the number of Transpose operations in the graph, for this purpose it moves Transposes through some operations (Convert in this case) and tries to fuse them. No precision checks are provided, the algo doesn't control it.

It looks like we need to change the approach to be able to detect the optimal position for Transpose op in the graph.

Reply:
@itikhono Thanks! In general case it is not that trivial to say what behavior would be more benefitial. Lets us analyze more cases before any adjustments.

@liubo-intel As for now I would still insist we shouldn't skip tests w/o significant reason. In this particular case lets update inType here https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/transpose.cpp#L56 for fp16 case -> it will change expected primitive type. Please don;t forget comment with description why custom behavior for fp16 is needed (regarding ov::pass::TransposeSinking behavior)

Reply:
Thanks @itikhono and @dmitry-gorokhov , 'change expected primitive type' indeed seems better than skip the whole test case currently. have changed to this way. 
BTW, it seems we should change both 'inType' and 'outType' for these special cases in order to change expected primitive type.

Reply:
when using this 'change expected primitive type' method, there are some test case failed with 'Expected and actual shape are different: [10,21,16,2] [2,16,21,10]'.
Thanks @itikhono help investigate, we found that these issues related with `[pass::EliminateTranspose::EliminateTranspose](https://github.com/openvinotoolkit/openvino/blob/2ec64af3509335b6e36eb8fe3171e8ea38526e59/src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp#L722)` eliminated this kind of 'empty order input' Transpose Node which is not expected. @itikhono will provide another pr to fix it.
before:
![1](https://github.com/openvinotoolkit/openvino/assets/56590087/9c3ad506-0a04-45bc-bde5-3bc3c0fb4d3a)
after:
![2](https://github.com/openvinotoolkit/openvino/assets/56590087/84143a08-93f7-4f38-96b2-bd59784fbe74)

Reply:
this is a PR with the fix https://github.com/openvinotoolkit/openvino/pull/24762

Reply:
Hi, @dmitry-gorokhov : based on your suggestion, have changed to the way: https://github.com/openvinotoolkit/openvino/pull/23950/commits/e08349a51e2c3932d49fe317b217ddb851a9ea6f